crates/oxc_diagnostics/Cargo.toml

```diff
@@ -13,7 +13,7 @@ categories.workspace   = true
 
 [dependencies]
 thiserror = { workspace = true }
-miette    = { workspace = true }
+miette    = { workspace = true, features = ["fancy"] }
```

Comment:
I had to add the feature "fancy" here to get fancy errors in `oxc_linter_plugin`'s tests, is there any better way to do this?

Reply:
Nope

Reply:
Add this as a dev-dependency to your crate (with the feature).

Reply:
Thanks!